__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const db = require("../db/connection");
+const app = require("../app");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+
+beforeEach(() => seed(data));
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("Testing endpoints in the app.js file", () => {
+  test("GET: 200 - request to api/topcs returns all available topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const topics = response.body.topics;
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              description: expect.any(String),
+              slug: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe("Error handling", () => {
+  test("GET: ", () => {});
+});
+
```

Comment:
My advise based on personal experience is that grouping tests by happy / sad path makes specific tests harder to find as your test suite expands. Having error tests together with happy path tests for one endpoint can feel like a nicer way to organise 👍